docs/live-preview-configs.md

```diff
@@ -132,13 +132,11 @@ The `stackSdk` object represents the `Stack` class that we get by executing the
 
 ## `onLiveEdit(callback: () => void)`
 
-The `onLiveEdit` method modifies or alters the content inside the Live Preview panel as soon as a change is made in the entry.
+The onLiveEdit method modifies or alters the content inside the Live Preview panel as soon as a change is made in the entry. This method runs a single API request to retrieve draft content from the entry and display the changes in the Live Preview panel.
 
-> **Note:** This function only works when [`ssr`](#ssr) is set to `false`, indicating that the application is of type [CSR](https://www.contentstack.com/docs/developers/set-up-live-preview/set-up-live-preview-for-your-website/#client-side-rendering-csr-).
+> **Note:** The onLiveEdit method will not fetch the published content of the entry and is only applicable in the Client-Side Rendering ([CSR](https://www.contentstack.com/docs/developers/set-up-live-preview/set-up-live-preview-for-your-website/#client-side-rendering-csr-)) mode.
```

Comment:
Did we get this from the docs team?

Reply:
Yes
https://docs.google.com/document/d/15vqTPuNTYnlETPkaLsyHNuiYBWzh3fu-quH351gzwi4/edit#